packages/core/test/deal-tracker-service.test.js

```diff
@@ -0,0 +1,58 @@
+import { test as filecoinApiTest } from '@web3-storage/filecoin-api/test'
+import { ed25519 } from '@ucanto/principal'
+
+import { createClient } from '../src/store/deal-store.js'
+import { dealStoreTableProps } from '../src/store/index.js'
+
+import { testService as test } from './helpers/context.js'
+import { createDynamodDb, createTable } from './helpers/resources.js'
+
+test.beforeEach(async (t) => {
+  const dynamo = await createDynamodDb()
+
+  Object.assign(t.context, {
+    dynamoClient: dynamo.client,
+  })
+})
+
+for (const [title, unit] of Object.entries(filecoinApiTest.service.dealTracker)) {
+  const define = title.startsWith('only ')
+    // eslint-disable-next-line no-only-tests/no-only-tests
+    ? test.only
+    : title.startsWith('skip ')
+    ? test.skip
+    : test
```

Comment:
Not blocking...but we should probably switch to `entail` for new work as we get this functionality for free, and also an `assert`. See the billing PR for example https://github.com/web3-storage/w3infra/pull/240

Reply:
I will leave this for past MVP to not expand on change set